cpu/cortexm_common/thread_arch.c

```diff
@@ -234,6 +238,15 @@ char *thread_arch_stack_init(thread_task_func_t task_func,
     return (char*) stk;
 }
 
+/* This function returns the number of bytes used on the ISR stack */
+int thread_arch_isr_stack_usage(void)
+{
+    uint32_t  *ptr = &_sstack;
+    while (*(ptr++) == STACK_CANARY_WORD) {
+    }
+    return ISR_STACKSIZE - (ptr - &_sstack);
```

Comment:
So this means that the number currently shown is 4 times larger than the actual usage on 32-bit platforms, right? This would explain the quite significant ISR stack usage I saw when testing this PR. It that's so, there is more more thing I don't need to investigate and cross of my list :-)